### PR TITLE
Support more cases from the iTop editor

### PR DESCRIPTION
**Pull request by Hipska**

### CODE REVIEW

**Comment by Hipska** [comment]
Hi @larhip @dw-itomig, could you please give feedback or merge?

**Comment by larhip** [comment]
Hi @Hipska,

due to the upcoming webhook implementation by Combodo, which will be also part of the product in the future, we decided to do no more further development for this extension.

As soon as there will be an official combodo version available we will describe how to migrate and archive this repository. Anyhow if you plan to use itomig-webhook-integration in the future instead of the combodo implementation, feel free to use this one as a base and do your further development within your fork.

**Comment by larhip** [comment]
Reopen this again based on discussion in #16

File: datamodel.itomig-webhook-integration.xml [hunk 1]
```diff
@@ -400,7 +400,8 @@
           {
             //convert html to slack markdown
             if(!empty($sText)){
-              $sText = strip_tags($sText, '<h1><h2><h3><br><strong><b><em><i><del><s><li><code><pre><a>');
+                  $sText = strip_tags($sText, '<h1><h2><h3><br><strong><b><em><i><var><cite><del><s><q><li><tt><code><kbd><samp><pre><a>');
+                  $sText = preg_replace('/<([b-z][a-z0-9]*)[^>]*?(?:\/?)>/', '<$1>', $sText);
```
Comment (larhip) [inline datamodel.itomig-webhook-integration.xml:404]: Just to make sure I understand it right: This one does remove all attritubes (such like classes and so on) from the tags, doesn't it?

**Comment by Hipska** [comment]
Hi @larhip any idea when you will merge and release new version?

File: datamodel.itomig-webhook-integration.xml (hunk 1 above)
Comment (larhip) [reply datamodel.itomig-webhook-integration.xml:404]: @Hipska

**Comment by larhip** [comment]
Hi @Hipska 
did review this some time ago and got a question but forget to mention you in this question yo you did not receive a notification. SInce I was waiting for a feedback this issue got a little bit lost on my todo list.

**Comment by Hipska** [comment]
Okay, what is the question then?

**Comment by larhip** [comment]
please see here https://github.com/itomig-de/itomig-webhook-integration/pull/15/files

**Comment by Hipska** [comment]
Yeah, these are my changes. What should I see there?

**Comment by larhip** [comment]
image

**Comment by Hipska** [comment]
Okay, so you have a pending comment, can you also actually submit it, so I can respond? (Click the finish review button)

**Comment by larhip** [comment]
Sorry @Hipska it seems like this was my incompetence of using github :(

File: datamodel.itomig-webhook-integration.xml (hunk 1 above)
Comment (Hipska) [reply datamodel.itomig-webhook-integration.xml:404]: it removes attributes indeed, except for the `<a>` tag.

**Pull request merged**